Tests/test_file_msp.py

```diff
@@ -24,14 +27,43 @@ def test_invalid_file(self):
         self.assertRaises(SyntaxError,
                           lambda: MspImagePlugin.MspImageFile(invalid_file))
 
-    def test_open(self):
+    def test_bad_checksum(self):
+        # Arrange
+        # This was created by forcing Pillow to save with checksum=0
+        bad_checksum = "Tests/images/hopper_bad_checksum.msp"
+
+        # Act / Assert
+        self.assertRaises(SyntaxError,
+                          lambda: MspImagePlugin.MspImageFile(bad_checksum))
+
+    def test_open_windows_v1(self):
         # Arrange
         # Act
         im = Image.open(TEST_FILE)
 
         # Assert
         self.assertEqual(im.size, (128, 128))
         self.assert_image_equal(im, hopper("1"), 4)
+        self.assertIsInstance(im, MspImagePlugin.MspImageFile)
+
+    @unittest.skipIf(not os.path.exists(EXTRA_DIR),
+                     "Extra image files not installed")
+    def test_open_windows_v2(self):
+        # Arrange
+        ImageFile.LOAD_TRUNCATED_IMAGES = True
```

Comment:
The files are truncated and otherwise give:
```
Traceback (most recent call last):
  File "Tests/test_file_msp.py", line 60, in test_open_windows_v2
    im.load()
  File "/usr/local/lib/python2.7/site-packages/PIL/ImageFile.py", line 226, in load
    "(%d bytes not processed)" % len(b))
IOError: image file is truncated (0 bytes not processed)
```
These are the only example files I can find on the internet. I can't find any other software to generate them, or open these, and I've only got Windows 7 not Windows 1 or 2 :)

`im.show()`ing them gives reasonable-looking images, although with a black bar below the main image. The size fetching inside MspImageFile.py is the same for both Windows 1 and 2 formats, which looks sound for Windows 1, but MspDecode.c is only used for Windows 2 to do the actual decoding.

I've added some `getpixel` assertions.